app/src/api/fetchutils.ts

```diff
@@ -0,0 +1,20 @@
+const BASE_API_URL = import.meta.env.VITE_API_URL;
+
+export async function get(url: string, body?: Record<string, unknown>) {
```

Comment:
We need to add generics so we're able to have typed data from this util, i.e. so we can do:

```Typescript
async function fetchUser(userId: number) {
  const user = await get<User>(`/users/${userId}`);

  return user; // typed as User
}
```

```suggestion
export async function get<T>(url: string, body?: Record<string, unknown>) {
  ...
    const data: T = await response.json();
    return data;
  ...
```